### PR TITLE
Fix the tests

### PR DESCRIPTION
**Pull request by peternewman**
For the new normalised hidden file behaviour, this shouldn't be merged until version 1.18.0 or greater is released.

### CODE REVIEW

**Comment by larsoner** [comment]
I see you requested review, feel free to ping me once things are green

**Comment by peternewman** [comment]
> I see you requested review, feel free to ping me once things are green

Yeah sorry I forgot/hadn't looked at the BATS stuff failing.

**Comment by peternewman** [comment]
Okay @larsoner all the required tests are green again now. I've left the bats diagnostic thing in as it can help in future and just left it as not required, but I could remote it completely for now if you'd rather?

File: test/test.bats [hunk 1]
```diff
@@ -3,12 +3,16 @@
...
+# This currently breaks the tests, so only enabling when troubleshooting
+#load teardown
+
 ROOT_MISSPELLING_COUNT=5
 FILENAME_MISSPELLING_COUNT=1
 HIDDEN_MISSPELLING_COUNT=1
 EXCLUDED_MISSPELLING_COUNT=1
 BUILTIN_NAMES_MISSPELLING_COUNT=1
-IGNORE_WORDS_MISSPELLING_COUNT=6
+IGNORE_WORDS_MISSPELLING_COUNT=5
```
Comment (peternewman) [inline test/test.bats:15]: Now we don't include hidden files in some cases

**Comment by larsoner** [comment]
> I've left the bats diagnostic thing in as it can help in future and just left it as not required, but I could remote it completely for now if you'd rather?

This makes it harder to review. I'd at least get it to "fail gracefully" so that things are green instead of part-red -- for example, returning a "neutral" rather than "failed" or "succeeded" status would be preferable

**Review by larsoner — APPROVED** [review]
Other than not liking the "red on some is okay to merge" LGTM

**Comment by peternewman** [comment]
> This makes it harder to review. I'd at least get it to "fail gracefully" so that things are green instead of part-red -- for example, returning a "neutral" rather than "failed" or "succeeded" status would be preferable

There's not really a neutral in GitHub Actions compared to Travis' Allowed Failures. Best bet would be as I'd done with it not being required so I've skipped it for now.

**Pull request merged**